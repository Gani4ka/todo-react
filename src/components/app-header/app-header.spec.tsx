import React from 'react';
import { shallow } from 'enzyme';
import AppHeader, { AppHeaderProps } from './app-header';

describe('<AppHeader />', (): void => {
  const props: AppHeaderProps = {
    done: 1,
    toDo: 1
  };
  it('should render corectly', (): void => {
    const component = shallow(<AppHeader {...props} />);

    expect(component.debug()).toMatchSnapshot();
  });

  it('should render props', (): void => {
    props.done = 6;
    props.toDo = 3;
    const component = shallow(<AppHeader {...props} />);

    expect(component.find('h2').text()).toEqual(
      `${props.toDo} more to do, ${props.done} done`
    );
  });
});
