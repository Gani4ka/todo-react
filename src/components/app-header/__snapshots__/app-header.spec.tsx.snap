// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppHeader /> should render corectly 1`] = `
"<div className=\\"app-header d-flex\\">
  <h1>
    Todo List
  </h1>
  <h2>
    1
     more to do, 
    1
     done
  </h2>
</div>"
`;
